packages/nextjs/src/tools/codegen/extract-files.ts

```diff
@@ -41,8 +40,8 @@ export const extractFiles = (args: ExtractFilesConfig) => {
     try {
       // MESH_URL is temporary option to use until mesh is onboarded into Edge Proxy
       const targetUrl = process.env.SITECORE_MESH_URL || args.scConfig.api.edge.edgeUrl;
-      const bearer = await fetchBearerToken(authParams);
-      if (!bearer) {
+      const { accessToken } = await auth.clientCredentialsFlow(authParams);
+      if (!accessToken) {
         console.error(chalk.red('Failed to get bearer token, aborting code extraction'));
```

Comment:
Should it be updated?
```suggestion
        console.error(chalk.red('Failed to get access token, aborting code extraction'));
```